stdlib/SparseArrays/src/sparsevector.jl

```diff
@@ -1676,8 +1676,6 @@ function densemv(A::AbstractSparseMatrixCSC, x::AbstractSparseVector; trans::Abs
         mul!(y, transpose(A), x)
     elseif trans == 'C' || trans == 'c'
         mul!(y, adjoint(A), x)
```

Comment:
```suggestion
    else # trans == 'C' || trans == 'c'
        mul!(y, adjoint(A), x)
```
By the same logic, it seems like we don't need any checks here anymore.